x/pairing/keeper/scores/stake_req.go

```diff
@@ -17,7 +17,11 @@ func (sr *StakeReq) Init(policy planstypes.Policy) bool {
 
 // Score calculates the the provider score as the normalized stake
 func (sr *StakeReq) Score(score PairingScore) math.Uint {
-	return sdk.NewUint(score.Provider.EffectiveStake().Uint64())
+	effectiveStake := score.Provider.EffectiveStake()
+	if effectiveStake.IsNegative() || effectiveStake.IsZero() {
```

Comment:
I think there's a `IsPositive()` method, if I'm not wrong

Reply:
done. see https://github.com/lavanet/lava/commit/1e2fd6253b305d69b704b5f57f3f6483601ac4af